Packs/cisco-meraki/Integrations/CiscoMerakiv2/CiscoMerakiv2.yml

```diff
@@ -0,0 +1,2283 @@
+    - contextPath: CiscoMeraki.SSID.Wireless.name
+      description: The name of the SSID.
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusAttributeForGroupPolicies
+      description: RADIUS attribute used to look up group policies enum = [Airespace-ACL-Name, Aruba-User-Role, Filter-Id, Reply-Message].
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusFailoverPolicy
+      description: Policy which determines how authentication requests should be handled in the event that all of the configured RADIUS servers are unreachable enum = [Allow access, Deny access].
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusLoadBalancingPolicy
+      description: Policy which determines which RADIUS server will be contacted first in an authentication attempt, and the ordering of any necessary retry attempts enum = [Round robin, Strict priority order].
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.splashPage
+      description: The type of splash page for the SSID enum = [Billing, Cisco ISE, Click-through splash page, Facebook Wi-Fi, Google Apps domain, Google OAuth, None, Password-protected with Active Directory, Password-protected with LDAP, Password-protected with Meraki RADIUS, Password-protected with custom RADIUS, SMS authentication, Sponsored guest, Systems Manager Sentry].
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.splashTimeout
+      description: Splash page timeout.
+      type: Date
+    - contextPath: CiscoMeraki.SSID.Wireless.wpaEncryptionMode
+      description: The types of WPA encryption enum = [WPA1 and WPA2, WPA1 only, WPA2 only, WPA3 192-bit Security, WPA3 Transition Mode, WPA3 only].
+      type: String
+    - contextPath: CiscoMeraki.SSID.Wireless.availableOnAllAps
+      description: Whether all APs broadcast the SSID or if it's restricted to APs matching any availability tags.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.enabled
+      description: Whether or not the SSID is enabled.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.localAuth
+      description: Extended local auth flag for Enterprise NAC.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.mandatoryDhcpEnabled
+      description: Whether clients connecting to this SSID must use the IP address assigned by the DHCP server.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusAccountingEnabled
+      description: Whether or not RADIUS accounting is enabled.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.radiusEnabled
+      description: Whether RADIUS authentication is enabled.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.ssidAdminAccessible
+      description: SSID Administrator access status.
+      type: Boolean
+    - contextPath: CiscoMeraki.SSID.Wireless.visible
+      description: Whether the SSID is advertised or hidden by the AP.
```

Comment:
This description is confusing.
I think it should be: Whether the SSID is advertised.

Reply:
changed